cloudapi/insights/client.go

```diff
@@ -58,6 +63,21 @@ type ClientTLSConfig struct {
 	CertFile string
 }
 
+// RetryConfig is the configuration for the client retries.
+type RetryConfig struct {
+	RetryableStatusCodes string
+	MaxAttempts          uint
+	PerRetryTimeout      time.Duration
+	BackoffConfig        BackoffConfig
+}
+
+// BackoffConfig is the configuration for the client retries using the backoff exponential with jitter algorithm.
+type BackoffConfig struct {
```

Comment:
nit: Prefix with `Client` to match other struct names.

Reply:
Done, thanks!